kratos/processes/simple_mortar_mapper_process.cpp

```diff
@@ -260,9 +259,7 @@ inline bounded_matrix<double, TNumNodes, TNumNodes> SimpleMortarMapperProcess<TD
     bounded_matrix<double, TNumNodes, TNumNodes> inv_matrix = ZeroMatrix(TNumNodes);
```

Comment:
call it InvertDiagonalMatrix, at least it is possible to understand what it does.

also i would make it to accept TMatrixType, as the only template parameter. You can get all the rest from that one.

Reply:
Ok